app/models/jobsModel.js

```diff
@@ -1,22 +1,59 @@
 const db = require('../database/dbConfig.js')
+const crypto = require('crypto')
 
 /**
  * Inserts a job into the database.
  * scheduled_time is an optional field.
  * 
  * @param {"class_id" : integer, 
- *         "scheduled_time" : Date, 
- *         "job_hash" : string} job 
+ *         "scheduled_time" : Date} job 
  */
 async function insertJob(job){
     try {
+        const dateString = (new Date()).valueOf().toString()
```

Comment:
Good job changing the format and style, two more thing
1. remember I have mentioned before that the model functions should be as simple as possible. it techinically shouldn't do any addition modification to the parameters. You should move these lines to the place before calling this function
2. make sure to use semi colons.